File: C++/Step2-If/9498.cpp
//
//  9498.cpp
//  Step2-If
//
//  Created by 김예빈 on 2019. 8. 28..
//  Copyright © 2019년 김예빈. All rights reserved.
//

#include <iostream>
using namespace std;

int main(int argc, const char * argv[]) {
    int n;
    
    cin >> n;
    
    if (n >= 90)
        cout << "A";
    else if (n >= 80)
        cout << "B";
    else if (n >= 70)
        cout << "C";
    else if (n >= 60)
        cout << "D";
    else
        cout << "F";
    
    return 0;
}

File: BaekJoon Online Judge/Swift/Step02-Conditional Statements/9498.swift
//
//  9498.swift
//  Step02-If
//
//  Created by Yebin Kim on 2019/10/27.
//  Copyright © 2019 김예빈. All rights reserved.
//

import Foundation

var score = Int(readLine()!)

if let score = score {
    if(score >= 90) {
        print("A")
    } else if(score >= 80) {
        print("B")
    } else if(score >= 70) {
        print("C")
    } else if(score >= 60) {
        print("D")
    } else {
        print("F")
    }
}
